docs/data-fetching.md

```diff
@@ -68,7 +68,7 @@ export async function getEdgeProps() {
 }
 ```
 
-There might be times when you **never want the page to be cached**. That's possible, too — just return `{ revalidate: 0 }` from `getEdgeProps` to tell Flareact to fetch a fresh page every single request.
+There might be times when you **never want the page to be cached**. That's possible, too — just return `{ revalidate: 0 }` from `getEdgeProps` to tell Flareact to fetch a fresh page every single request.
```

Comment:
What is changing in this diff?

Reply:
I knew you would ask 🙈  I don't know why it's taking the blank space that we already had differently.